lib/command/cleanup_stale_apps.rb

```diff
@@ -72,18 +72,8 @@ def confirm_delete
       Shell.confirm("\nAre you sure you want to delete these #{stale_apps.length} apps?")
     end
 
-    def delete_gvc(app)
-      step("Deleting app '#{app[:name]}'") do
-        cp.gvc_delete(app[:name])
-      end
-    end
-
-    def delete_images(app)
-      app[:images].each do |image|
-        step("Deleting image '#{image}'") do
-          cp.image_delete(image)
-        end
-      end
+    def delete_app(app)
+      Cpl::Cli.start(["delete", "-a", app, "--yes"])
```

Comment:
Simply call the `delete` command instead of implementing its own delete methods.

It's ok to pass `--yes` here because this is only called after `confirm_delete` (https://github.com/shakacode/control-plane-flow/pull/175/files#diff-1070cffdca466619e96b18a62aa7a9c142379e8cf0ca9680455ea0dae7017e1eR28), so the confirmation step will already have been handled by the time we get here.